commands/eval.b.inc

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @file
+ * Command(s) for evaluating PHP code directly.
+ */
+
+/**
+ * Implements hook_b_command().
+ */
+function eval_b_command() {
+  return array(
+    'php-eval' => array(
+      'description' => bt('Evaluate arbitrary PHP code after bootstrapping Backdrop.'),
```

Comment:
Remove reference to bootstrapping Backdrop (see note below about bootstrap). Also explain 'evaluate', since it sounds like this command is meant to *review* PHP code...

```suggestion
      'description' => bt('Evaluate (run/execute) arbitrary PHP code.'),
```

Reply:
I copied the description and bootstrap level directly from the equivalent drush command, figuring that people moving from drush to b would expect to see the same description for the same commands that they were used to. But if your philosophy is to reword like this, I'm fine with the change.

Reply:
Per the next comment: since Backdrop is going to be fully bootstrapped, and that's an important property that users of this command will be relying on, it seems reasonable to mention it in the description.

Reply:
Let me also address the question of whether "(run/execute)" is needed. I am not particularly opposed to adding the parenthetical, but I do note that "evaluate" is the terminology used [in the PHP manual for this action](https://www.php.net/manual/en/function.eval).

Reply:
As per my comment below, I agree that we need Backdrop bootstrapped for this command, so that text can stay in the description. However I still see the benefit of the bracketed text, so I'd like that added.

```suggestion
      'description' => bt('Evaluate (run/execute) arbitrary PHP code after bootstrapping Backdrop.'),
```

Reply:
OK, will add.